added article on Responsibility (#296)
DXHeroes/knowledge-base-content: practices/responsibility.md

  - ability
  - response
  - appreciation
  - problems
  - problem-solving
  - assign
  - leader

keywords:
  - responsibility
  - trust
  - people
  - benefits
  - team lead
---

# Responsibility

**TL;DR**
Responsibility is about trusting people and their commitment towards the work assigned. A responsible person completes the task assigned to them no matter what the circumstances are.

## What Is Responsibility

**Response + ability**, these two words combine to form a well known word **Responsibility**. It means the ability of a person to respond to a situation according to its capabilities. In tough situations, a responsible person will never shy away from its responsibilities, instead it will rise to the occasion to save the sinking ship. Taking responsibility requires a great sense of courage as well because if the person fails he or she not only fails themselves but also the entire team that are dependent on them. However, taking responsibility beyond oneself means a greater level of risk and therefore a greater degree of success.

Responsibility is not for everyone but who chooses to become responsible for their actions, achieves a greater sense of power and freedom.

> ![Responsibility](/files/responsibility.jpg)

Reviewer: Please, add the source of your image.

## Why You Might Want Responsibility

Responsibility brings us many folds of its benefits. It helps an individual achieve its goals and complete the assigned work before deadline. It helps the company or oganization work more efficiently and keep its client happy.

From a team point of view, if we have more responsible people in our team, the better will be our performance: 

- Each team member will be able to work more independently and make better decisions. 
- We can trust the team members about the work assigned to them.
- The team members will also tend to be more honest and keep their promises.

> - It will create a nice environment for others members to work.

Reviewer: ```suggestion
- It will create a nice environment for others members to work.
- Responsibility is part of healthy [team culture]()
```
One great guy is working on team culture article. If you won't mind, I would suggest adding it here.

  - ability
  - response
  - appreciation
  - problems
  - problem-solving
  - assign
  - leader

keywords:
  - responsibility
  - trust
  - people
  - benefits
  - team lead
---

# Responsibility

**TL;DR**
Responsibility is about trusting people and their commitment towards the work assigned. A responsible person completes the task assigned to them no matter what the circumstances are.

## What Is Responsibility

**Response + ability**, these two words combine to form a well known word **Responsibility**. It means the ability of a person to respond to a situation according to its capabilities. In tough situations, a responsible person will never shy away from its responsibilities, instead it will rise to the occasion to save the sinking ship. Taking responsibility requires a great sense of courage as well because if the person fails he or she not only fails themselves but also the entire team that are dependent on them. However, taking responsibility beyond oneself means a greater level of risk and therefore a greater degree of success.

Responsibility is not for everyone but who chooses to become responsible for their actions, achieves a greater sense of power and freedom.

> [](/files/responsibility.jpg)

Reviewer 2: ```suggestion
[Responsibility](/files/responsibility.jpg)
```

- The product a team is working is successful.
- [No toxic team culture](/problems/toxic-team-culture)

## How to Implement Responsibility

- **Communication is the key**: The first step to implementing responsibility is talking to the concerned individual about their responsibility.
- **Enough resources to complete their work**: The people must be given adequate resources that they might need to complete their work. This might include proper training, guidance, access to resources, etc. | [Home Office](/practices/home-office)
- **Responsibilities and Clearity**: The people must we well aware about their responsibiltes and have a sense of clearity about the tasks they have to perform. | [Team Roles Definition](/practices/team-roles-definition)
- **Re-assign**: There might me times when a person is not fit to be assigned a particular task so at those moments the responsibility must be re-assigned to a better person.
- **Reflection Meetings**: The team members must have short and insightful meetings for reflection about their work done and to discuss future plans | [Meaningful Meetings](/practices/meaningful-meetings) | [Retrospective](/practices/retrospective)

> ![Meetings](/files/responsibility2.jpg)

Reviewer 2: ```suggestion

![Meetings](/files/responsibility2.jpg)
```

> [](/files/responsibility2.jpg)

Reviewer 2: ```suggestion

[Responsibility](/files/responsibility2.jpg)
```

  - ability
  - response
  - appreciation
  - problems
  - problem-solving
  - assign
  - leader

keywords:
  - responsibility
  - trust
  - people
  - benefits
  - team lead
---

# Responsibility

**TL;DR**
Responsibility is about trusting people and their commitment towards the work assigned. A responsible person completes the task assigned to them no matter what the circumstances are.

## What Is Responsibility

**Response + ability**, these two words combine to form a well known word **Responsibility**. It means the ability of a person to respond to a situation according to its capabilities. In tough situations, a responsible person will never shy away from its responsibilities, instead it will rise to the occasion to save the sinking ship. Taking responsibility requires a great sense of courage as well because if the person fails he or she not only fails themselves but also the entire team that are dependent on them. However, taking responsibility beyond oneself means a greater level of risk and therefore a greater degree of success.

Responsibility is not for everyone but who chooses to become responsible for their actions, achieves a greater sense of power and freedom.

> [Responsibility](/files/responsibility.jpg)
> ![Responsibility](https://shiftyourfamilybusiness.com/wp-content/uploads/2018/05/ResponsibilityBlog-845x321.jpg)

Reviewer 2: ```suggestion
![Responsibility](/files/responsibility.jpg)
[Your Response Is Your Responsibility](https://shiftyourfamilybusiness.com/2018/05/20/your-response-is-your-responsibility/)
```

## Why You Might Want Responsibility

Responsibility brings us many folds of its benefits. It helps an individual achieve its goals and complete the assigned work before deadline. It helps the company or oganization work more efficiently and keep its client happy.

From a team point of view, if we have more responsible people in our team, the better will be our performance: 

- Each team member will be able to work more independently and make better decisions. 
- We can trust the team members about the work assigned to them.
- The team members will also tend to be more honest and keep their promises.

> - Responsibility is a part of healthy [team culture](https://preview.developerexperience.io/practices/team-culture)

Reviewer 2: ```suggestion
- Responsibility is a part of healthy [team culture](/practices/team-culture)
```

## Problems Responsibility Solves

- Creates a positive atmosphere.
- Matches the expectations of the company.
- Increases the self worth of an individual, as the work done by them is appreciated.

> - Quality of work is higher and no poor quality work is done | [Poor Code Quality](https://developerexperience.io/problems/poor-code-quality)

Reviewer 2: ```suggestion
- Quality of work is higher and no poor quality work is done | [Poor Code Quality](/problems/poor-code-quality)
```

> - Team focuses mostly on solving problems rather than indulging in any unhealtly blame game | [Demotivated Team](https://developerexperience.io/problems/demotivated-team)

Reviewer 2: ```suggestion
- Team focuses mostly on solving problems rather than indulging in any unhealthy blame game | [Demotivated Team](/problems/demotivated-team)
```

- The product a team is working is successful.

> - [No toxic team culture](https://developerexperience.io/problems/toxic-team-culture)

Reviewer 2: ```suggestion
- [No toxic team culture](/problems/toxic-team-culture)
```

## How to Implement Responsibility

- **Communication is the key**: The first step to implementing responsibility is talking to the concerned individual about their responsibility.

> - **Enough resources to complete their work**: The people must be given adequate resources that they might need to complete their work. This might include proper training, guidance, access to resources, etc. | [Home Office](https://developerexperience.io/practices/home-office)

Reviewer 2: ```suggestion
- **Enough resources to complete their work**: The people must be given adequate resources that they might need to complete their work. This might include proper training, guidance, access to resources, etc. | [Home Office](/practices/home-office)
```

> - **Responsibilities and Clearity**: The people must we well aware about their responsibiltes and have a sense of clearity about the tasks they have to perform. | [Team Roles Definition](https://developerexperience.io/practices/team-roles-definition)

Reviewer 2: ```suggestion
- **Responsibilities and Clearity**: The people must we well aware about their responsibiltes and have a sense of clearity about the tasks they have to perform. | [Team Roles Definition](/practices/team-roles-definition)
```

- **Re-assign**: There might me times when a person is not fit to be assigned a particular task so at those moments the responsibility must be re-assigned to a better person.

> - **Reflection Meetings**: The team members must have short and insightful meetings for reflection about their work done and to discuss future plans | [Meaningful Meetings](https://developerexperience.io/practices/meaningful-meetings) | [Retrospective](https://developerexperience.io/practices/retrospective)

Reviewer 2: ```suggestion
- **Reflection Meetings**: The team members must have short and insightful meetings for reflection about their work done and to discuss future plans | [Meaningful Meetings](/practices/meaningful-meetings) | [Retrospective](/practices/retrospective)
```

> [Responsibility](/files/responsibility2.jpg)
> ![Meetings](https://blog.bonus.ly/hubfs/productive-enjoyable-meetings.jpg)

Reviewer 2: ```suggestion
![Meetings](/files/responsibility2.jpg)
[How to Hold Productive and Enjoyable Meetings](https://blog.bonus.ly/productive-and-enjoyable-meetings)
```

## Common Pitfalls of the Responsibility

> - People run from responsibilities for reasons like low self-worth and fear | [Toxic Team Culture](https://developerexperience.io/problems/toxic-team-culture)

Reviewer 2: ```suggestion
- People run from responsibilities for reasons like low self-worth and fear | [Toxic Team Culture](/problems/toxic-team-culture)
```

---
slug: responsibility
stages:
  - development

> short_description: Responsibility is keeping up the trust and faith of people who trusts you, no matter what the situation is. Responsibility primarily involves two things -- authority and accountability; however, trust is often a factor as well.

Reviewer: Our plagiarism check found similar text to this. It is on Quora (https://www.quora.com/What-is-responsibility) and it is from Abhimanyu Bellam: _Responsibility is keeping up the trust and faith of people (who have trust on u) no matter what the situation is._

Other similarity can be found with Quora (https://www.quora.com/How-would-you-define-responsibility) text from James Longwell: _Responsibility primarily involves two things: authority and accountability; however, trust is often a factor as well_

Author: Hi, 
Thank you so much for reviewing this and suggesting me the changes. First of all, I am extremely sorry for increasing your work and after checking all the suggested changes, I am going to re-write the entire article on my own.
Kindly, give me few days to create the article once again.

> slug: responsibility
> stages:
>   - development
>  short_description: Responsibility is keeping up the trust and faith of people who trusts you, no matter what the situation is. Responsibility primarily involves two things -- authority and accountability; however, trust is often a factor as well.
> 
>  tags:
>     - responsibility
>     - team work
>     - ability
>     - response
>     - appreciation
>     - problems
>     - problem-solving
>     - assign
>     - leader
> 
>  keywords:
>     - responsibility
>     - trust
>     - people
>     - benefits
> ---

Reviewer 2: Fix formatting of meta info
```suggestion
slug: responsibility
stages:
  - development
short_description: Responsibility is keeping up the trust and faith of people who trusts you, no matter what the situation is. Responsibility primarily involves two things -- authority and accountability; however, trust is often a factor as well.

tags:
  - responsibility
  - team work
  - ability
  - response
  - appreciation
  - problems
  - problem-solving
  - assign
  - leader

keywords:
  - responsibility
  - trust
  - people
  - benefits
---
```

# Responsibility

**TL;DR**

Responsibility is keeping up the trust and faith of people who trusts you, no matter what the situation is.

## What Is Responsibility

> Responsibility is the combination of two words: **Response + ability**. It also means taking on more risk and therefore reward because it involves a degree of success and failure beyond oneself. People are responsible for themselves and so when failure happens, you are just failing yourself. When you choose to take on responsibilities beyond yourself, you are agreeing to take the interests of others into your hands in trying to do more, be more, and allow others to benefit from your being responsible.

Reviewer: Unfortunately, our plagiarms check found similar text on Quora (https://www.quora.com/What-is-responsibility). The text in questions is from Blair Capel: _Responsibility means taking on more risk and therefore reward because it involves a degree of success and failure beyond oneself. People are responsible for themselves and so when failure happens, you are just failing yourself. When you choose to take on responsibilities beyond yourself, you are agreeing to take the interests of others into your hands in trying to do more, be more, and allow others to benefit from your being responsible.
Responsibility is not for everyone. But it’s a powerful and deeply rewarding ship to navigate once embarked._

Responsibility is not for everyone. But it’s a powerful and deeply rewarding ship to navigate once embarked.

![Responsibility](https://shiftyourfamilybusiness.com/wp-content/uploads/2018/05/ResponsibilityBlog-845x321.jpg)

## Why You Might Want Responsibility

> Being responsible brings us many benefits. It can help us achieve our goals and objectives in any area of our life. Responsibility allows us to create principles, morals and helps us to lead your life. Being a responsible person helps us to:

Reviewer: This part has similarities with _Becoming a responsible person means being able to consciously make decisions, conduct behaviors that seek to improve oneself and/or help others. Most importantly, a responsible person accepts the consequences of his or her own actions and decisions._ from https://blog.cognifit.com/responsibility/

> **Be more independent:** Assuming the consequences of our actions will help us make better decisions.

Reviewer: From the cognifit article: _Be more independent: Assuming the consequences of our actions will help us make better decisions._

Please, rewrite this part.

> **Be more reliable**: By being responsible, we gain other's trust and we will also gain confidence in ourselves

Reviewer: From the cognifit article:_Be more reliable: By being responsible, we gain other’s trust and we will also gain confidence in ourselves. Doing the right thing will make us feel good. And even if we are wrong, we will be satisfied because we know that we have done our best._

Could you please rewrite this with your own words?

> **Be more honest:** When we tend to tell the truth and keep our promises, the people around us will believe us and see us as an honest person.

Reviewer: From the same cofnigit article: _Be more honest: When we tend to tell the truth and keep our promises, the people around us will believe us and see us as an honest person._

Please, give it a second look.

## Problems Responsibility Solves

- Establishes team accountability
- Key to great team work
- Quality of output is usually high as the tasks done by the team are mostly interdependent
- An individual is committed to complete the tasks allocated to them
- Problem solving is given importance rather than blaming others
- Makes your team members feel important and appreciated
- Helps an individual do more, faster

## How to Implement Responsibility

> 

Reviewer: I really dig the following paragraphs. Do you think you could include some of the other practices DX Knowledgebase (https://developerexperience.io/search?q=team+culture) that cover "similar" things?

- **Start by Talking**: The first step is to talk to the individuals concerned

> - **Ensure Adequate Resources**: A sensible action is to ensure that your people have the resources they need to do their job . This might include providing training, equipment, access to mentors  and coaches , and suchlike.

Reviewer: I am sorry, our plagiarism check found the same text in a mindtools article. The mentioned text is: _A sensible early action is to ensure that your people have the resources they need to do their job  . This might include providing training, equipment, access to mentors and coaches, and suchlike._ Could you please rewrite it or rephrase it?

> - **Communicate Roles, Responsibilities, and Objectives**: Your people also need to know clearly what their job roles and responsibilities are.

Reviewer: I am sorry, there is a striking similarity with _Communicate Roles, Responsibilities, and Objectives Your people also need to know clearly what their job roles and responsibilities are._  Could you please rewrite it with your own words?

> - **Re-Engage People**: You take responsibility for your actions, simply because you have a deep sense of pride in what you're doing. The same will likely hold true for your people: by working on [re-engaging](https://www.mindtools.com/pages/article/newTMM_83.htm) them.

Reviewer: I am sorry, our plagiarism check found the same text in a mindtools article. The mentioned text is
_You take responsibility for your actions, simply because you have a deep sense of pride in what you're doing. The same will likely hold true for your people: by working on re-engaging  (same link) them._

Could you please rephrase this?

**Tips for implementing Responsibility**

> 1. Make sure that you have clear, accurate examples that you can cite when you provide feedback. 

Reviewer: Could you give this a second look? It seems too close to: _Make sure that you have clear, accurate examples that you can cite when you provide feedback._ from the mindtools article.

> 2. Take the [How Well Do You Develop Your People?](https://www.mindtools.com/pages/article/team-development.htm)  self-test to improve your team development skills.

Reviewer: This is similar to: _Take our How Well Do You Develop Your People?   self-test to improve your team development skills._

> 3. Sometimes, people don't take responsibility because they feel apathetic about their work. They can't see how their efforts tie into the "bigger picture." So, make sure that they understand how their work ties into the larger goals of the organization

Reviewer: Please, rewrite this sentence since it is similar to a sentence from the Mindtools article: _Sometimes, people don't take responsibility because they feel apathetic about their work. They can't see how their efforts tie into the "bigger picture."_

> 4. People who don't take responsibility often play the blame game. When you notice team members starting to point the finger of blame, stop them immediately. 

Reviewer: I am deeply sorry, but could you rewrite this sentence as well? It is too close to the original one (_People who don't take responsibility often play the blame game. When you notice team members starting to point the finger of blame, stop them immediately._)

> 5. Be sure to give your people plenty of praise when they do take responsibility. And, help them improve by providing them with consistent, effective, fair feedback.

Reviewer: Similarly, this sentence doesn't pass our plagiarism check. Consider rewriting it: _Finally, be sure to give your people plenty of praise when they do take responsibility. And, help them improve by providing them with consistent, effective, fair feedback._

## Common Pitfalls of the Responsibility

> - People duck responsibilities for reasons like laziness

Reviewer: Seems to close to _People duck responsibility for reasons ranging from simple laziness (..)_

- An individual might have a fear of failure

> - A sense of feeling overwhelmed by the scale of a problem or a situation.

Reviewer: From the same article, _sense of feeling overwhelmed by the scale of a problem or a situation._

- Missing deadlines

> - Lacking interest in their work, and in the well-being of the team.

Reviewer: From the mindtools article: _Lacking interest in their work, and in the well-being of the team._

## Resources for Responsibility

> 

Reviewer: ```suggestion
[Helping People Take Responsibility](https://www.mindtools.com/pages/article/taking-responsibility.htm)

```

Reviewer: Do you think this and the next resource are good points for further reading?

> 

Reviewer: ```suggestion
- [Responsibility](https://blog.cognifit.com/responsibility/)
```
